tests/gamma/affinity_session_drain_test.py

```diff
@@ -53,7 +53,10 @@ class AffinitySessionDrainTest(  # pylint: disable=too-many-ancestors
     @staticmethod
     @override
     def is_supported(config: skips.TestConfig) -> bool:
-        if config.client_lang == _Lang.CPP and config.server_lang == _Lang.CPP:
+        if (
+            config.client_lang == _Lang.CPP
+            or config.client_lang == _Lang.PYTHON
+        ) and config.server_lang == _Lang.CPP:
```

Comment:
Wait, it won't ever pass this check unless we override the python client to user the server, similar to this:

https://github.com/grpc/psm-interop/blob/1544fc29e9c87b9d9a0f0f8e18c0babf9600051d/tests/affinity_test.py#L46-L52

Reply:
Interesting, I triggered a manual run and looks like it did pass with C++ server image and Python client image: https://btx.cloud.google.com/invocations/60ee9457-6c4a-4c13-ab01-f8370d499cca/targets.

Maybe because we're using C++ image in `psm-csm-python.sh`?
https://github.com/grpc/grpc/blob/b4bf9211173c4726012d6ed9ddf43cec834e2527/tools/internal_ci/linux/psm-csm-python.sh#L23

Reply:
Yea, makes sense then. But is this what we want to do?

Reply:
Synced offline, will change to use both Python client and server for testing (except session drain test).